internal/publish/publish_v1.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package publish defines the exposure keys publishing API.
+package publish
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"go.opencensus.io/trace"
+
+	"github.com/google/exposure-notifications-server/internal/jsonutil"
+	"github.com/google/exposure-notifications-server/internal/serverenv"
+	v1 "github.com/google/exposure-notifications-server/pkg/api/v1"
+	"github.com/mikehelmick/go-chaff"
+)
+
+// NewV1Handler creates the HTTP handler for the TTK publishing API set at v1.
```

Comment:
I feel like this would be better architected as a `type server struct` that has two hanging handlers - `handlev1` and `handlev1alpha` which return `http.Handler`.

Reply:
refactored this.